src/Membrain/Units.hs

```diff
@@ -96,7 +96,7 @@ UnitSymbol Byte :: Symbol
 = "B"
 >>> :kind! UnitSymbol Kilobyte
 UnitSymbol Kilobyte :: Symbol
-= "KB"
+= "kB"
```

Comment:
Our unit tests and `doctest` really paid off 😎 I'm happy with the testing we did!